ckanext/multilang/templates/package/snippets/resource_item_multilang.html

```diff
@@ -0,0 +1,10 @@
+{% extends 'package/snippets/resource_item.html' %}
+
+
+{% block resource_item_description %}
+    {{ super() }}
+
+    {% if res.lang %}
+        <p> {% trans %}Available in language{% endtrans %}: <span class="text text-info small">{{ h.get_lang_name(res.lang) }} [{{ res.lang }}]</span></p>
```

Comment:
@cezio why extract_messages did not work correclty?

Reply:
No idea why. Maybe extractor didn't scan all directories. I used setup.cfg similar to ones used in other extensions, so I assumed it will work it this case too.

Reply:
@cezio now if I'm a guest I cannot see the resources

![res_pr](https://user-images.githubusercontent.com/1280027/38603806-e588f9de-3d6e-11e8-8f3c-680172c55643.png)

Reply:
@cezio we should also allow a default empty value for the languange selector in resource form